src/main.rs

```diff
@@ -157,13 +160,14 @@ const DEFAULT_TEST_INDEX_HTML: &'static str = "
         Module['print'] = __cargo_web.print;
         Module['printErr'] = __cargo_web.print;
         Module['onExit'] = __cargo_web.on_exit;
+        Module['arguments'] = [{{#each arguments}} "{{this}}", {{/each}}];
```

Comment:
Unfortunately this seems broken? It emits `Module['arguments'] = [ "[object]", ];`